<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Publicador</class>
 <widget class="QWidget" name="Publicador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FotoYa</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lClave">
       <property name="text">
        <string>Clave</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lUsuario">
       <property name="text">
        <string>Usuario</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pbIngresar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ingresar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="leClave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>f0t0y400</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="leUsuario">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>juangabrielaguilera@outlook.com</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWebView" name="wv">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gbAlbum">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Crear álbum</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lNombreAlbum">
        <property name="text">
         <string>Nombre del álbum</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lDescrAlbum">
        <property name="text">
         <string>Descripción</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="leDescrAlbum"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="leNombreAlbum"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pbPublicarAlbum">
        <property name="text">
         <string>Publicar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leNombreAlbum</tabstop>
  <tabstop>leDescrAlbum</tabstop>
  <tabstop>pbPublicarAlbum</tabstop>
  <tabstop>pbIngresar</tabstop>
  <tabstop>leClave</tabstop>
  <tabstop>leUsuario</tabstop>
  <tabstop>wv</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
